### PR TITLE
New platforms

### PR DESCRIPTION
**Pull request by cw-dvr8**
Add biocratesP180 and BGISEQ-500 to platform.  Schema was compiled using ajv compile -s and is valid.  Refer questions to @Aryllen .

### CODE REVIEW

**Comment by karawoo** [comment]
This looks like it has some overlap with new assays in #659, but one is spelled differently ("Biocrates p180" vs. "biocratesP180") -- I don't feel strongly about which we use, but I think we should be consistent

File: terms/experimentalData/platform.json [hunk 1]
```diff
@@ -237,6 +237,16 @@
             "const": "Q Exactive",
             "description": "Thermo Scientific Q Exactive.",
             "source": "http://purl.obolibrary.org/obo/MS_1001911"
+        },
+        {
+            "const": "biocratesP180",
```
Comment (Aryllen) [inline terms/experimentalData/platform.json:242]:
```suggestion
            "const": "Biocrates p180",
```

**Review by Aryllen — CHANGES_REQUESTED** [review]
@karawoo, that's a good point.

@cw-dvr8, I think I would lean towards "Biocrates p180" for a couple reasons (I won't list them here unless it's needed).

File: terms/experimentalData/platform.json (hunk 1 above)
Comment (cw-dvr8) [reply terms/experimentalData/platform.json]: Updated.

**Review by Aryllen — APPROVED** [review]
Thanks, @cw-dvr8!

**Pull request merged**